vmm/src/vm.rs

```diff
@@ -419,6 +454,12 @@ impl Vm {
             &cmdline_cstring,
         )
         .map_err(Error::LoadCmdLine)?;
+
+        let initramfs_config = match self.initramfs {
+            Some(_) => Some(self.load_initramfs(mem.deref())?),
+            None => None,
+        };
+
```

Comment:
We need to change the `load_kernel()` routine a bit so that we decouple the kernel+cmdline loading from the `configure_system` call, otherwise we end up having to load an initramfs image in `load_kernel()`.

I'll open an issue about it and will handle it separately, you don't need to change that part of your PR.

Reply:
See issue #885